src/util/FioAddressUtils.ts

```diff
@@ -356,10 +360,28 @@ const updatePublicAddresses = async (
   try {
     edgeTx = await fioMakeSpend(fioWallet, action, {
       fioAddress,
-      publicAddresses
+      publicAddresses: publicAddresses.map(p => {
+        const fioAssets = infoServerData.rollup?.fioAssets ?? FIO_ASSET_MAP
+        const pluginId = Object.keys(fioAssets).find(pid => fioAssets[pid].chainCode === p.chain_code)
+
+        let parsedTokenCode = p.token_code.replace('.', '')
+
+        if (pluginId != null) {
+          const fioAsset = fioAssets[pluginId]
+          const tokenId = getTokenId(account.currencyConfig[pluginId], p.token_code)
+
+          parsedTokenCode = tokenId == null ? parsedTokenCode : fioAsset.tokenCodes[tokenId] ?? parsedTokenCode
```

Comment:
```
const tokenCode = fioAsset.tokenCodes[tokenId]
if (tokenId != null && tokenCode != null) {
  parsedTokenCode = tokenCode
}
```
Seems like we're casing the assignment statement under these conditions, so why not just put it in a code block me thinks.

Reply:
It's a weird statement because tokenId is possibly undefined.